### ISSUE TITLE
why i can't reproduce the effect with provided trained model

### ISSUE THREAD

**Issue by liuiyilang** [alterzero/DBPN-Pytorch#19]
I try to test the model on Set 5 X4 with provided trained models.When i calculate the psnr in RGB channel i got psnr =30.xxx  which seems not right.When i calculate the psnr in Y channel,i got psnr = 31.xxx. Both of them seems lower than psnr presented in the paper(32.47)。Is there anyone got the results presented in the paper?

**Comment by qingchuanhuajuan** [comment]
Run python3 eval.py, no SR picture. Only the following sentences, can you achieve it? What do I need to change?

Namespace(chop_forward=True, gpu_mode=True, gpus=1, input_dir='Input', model='models/DBPN_x4.pth', model_type='DBPNL', output='Results/', seed=123, self_ensemble=True, testBatchSize=1, test_dataset='Set5_LR_x4', threads=1, upscale_factor=4)
===> Loading datasets
===> Building model
Pre-trained SR model is loaded

**Comment by qingchuanhuajuan** [comment]
There are several models in the paper. Perhaps the X4 model that the author gives is not the best one. just guess.

**Comment by iRmantou** [comment]
@liuiyilang when i just use trained model weights "DBPN_x2.pth", I get wrong result, the result picture is green, did u get normall result?

**Comment by liuiyilang** [comment]
> @liuiyilang when i just use trained model weights "DBPN_x2.pth", I get wrong result, the result picture is green, did u get normall result?

I got the normal results ,but seems the model doesn't performs that good with psnr.

**Comment by liuiyilang** [comment]
> Run python3 eval.py, no SR picture. Only the following sentences, can you achieve it? What do I need to change?
> 
> Namespace(chop_forward=True, gpu_mode=True, gpus=1, input_dir='Input', model='models/DBPN_x4.pth', model_type='DBPNL', output='Results/', seed=123, self_ensemble=True, testBatchSize=1, test_dataset='Set5_LR_x4', threads=1, upscale_factor=4)
> ===> Loading datasets
> ===> Building model
> Pre-trained SR model is loaded

Sorry i can't help you,because i use my own dataloader and test code,while i just use dbpn.py and base_networks.py provided.

**Comment by iRmantou** [comment]
@liuiyilang thx, I have fixed it

**Comment by yangyingni** [comment]
> I try to test the model on Set 5 X4 with provided trained models.When i calculate the psnr in RGB channel i got psnr =30.xxx  which seems not right.When i calculate the psnr in Y channel,i got psnr = 31.xxx. Both of them seems lower than psnr presented in the paper(32.47)。Is there anyone got the results presented in the paper?

Have you dealt with it?And what's the PSNR of your result?Thank you.
